teams.json

```diff
@@ -17,20 +17,40 @@
       "name": "code-build-run",
       "apis": [
         "appengine",
+        "artifactregistry",
         "cloudbuild",
         "cloudfunctions",
         "cloudscheduler",
         "cloudtasks",
         "container",
-        "containeranalysis",
         "run",
         "logging",
-        "clouderrorreporting"
+        "clouderrorreporting",
+        "containeranalysis",
+        "containerregistry"
       ],
       "repos": [
         "googleapis/nodejs-logging-winston",
         "googleapis/nodejs-logging-bunyan",
+        "googleapis/nodejs-cloudbuild",
+        "googleapis/java-cloudbuild",
+        "googleapis/python-cloudbuild",
+        "googleapis/java-containeranalysis",
+        "googleapis/python-containeranalysis",
+        "googleapis/google-cloud-php-container-analysis",
```

Comment:
Due to the comment above, this can be deleted.  But another fun fact - PHP split repositories aren't quite the same as the others.  This is a readonly repo used as a publishing point because composer requires a standalone repository.  All issues are filed against `google-cloud-php`.